f2/src/components/button/__tests__/Button.test.js

```diff
@@ -1,42 +1,23 @@
 import React from 'react'
 import { render, cleanup } from '@testing-library/react'
 import { ThemeProvider } from 'emotion-theming'
-import theme from '../../../theme'
+import canada from '../../../theme/canada'
 import { Button } from '../'
 
+import chroma from 'chroma-js'
+
 describe('<Button />', () => {
   afterEach(cleanup)
 
-  it('properly renders child components', () => {
-    const { getAllByText } = render(
-      <ThemeProvider theme={theme}>
-        <Button>
-          <em>foo</em>
-        </Button>
-      </ThemeProvider>,
-    )
-    expect(getAllByText(/foo/)).toHaveLength(1)
-  })
-
-  it('uses styling props correctly', () => {
+  it('passes WCAG color contrast', () => {
```

Comment:
I'm not entirely sure what this test is testing - looks like we render the button, then compare black and white. We probably should do automated a11y testing differently than this - probably by relying more on our a11y test github action.